docs/usage/shooted_seed.md

```diff
@@ -0,0 +1,73 @@
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Project
+metadata:
+  name: garden
+spec:
+  owner:
+...
+  namespace: garden
+```
+
+## Create shoot
+
+### Deploy shoot
+
+Change the networks for the shooted seed that it is not overlapping with his shoots (spec.provider.infrastructureConfig.networks and spec.networking).
+Fullexample: [/example/90-shoot.yaml](../../example/90-shoot.yaml)
+
+Set the annotation for the shooted seed.
+Example:
+
+```yaml
+  annotations:
+    shoot.gardener.cloud/use-as-seed: >-
+      true,shootDefaults.pods=100.96.0.0/11,shootDefaults.services=100.64.0.0/13,disable-capacity-reservation,with-secret-ref
+```
+
+This annotation contains a few configurations for the seed:
+Option | Description
+--- | ---
+`true` | deploys the gardenlet into the shoot which will automatically register the cluster as seed
+`no-gardenlet` | prevents the automatic deployment of the gardenlet into the shoot. Instead, the `Seed` object will be created with the assumption that another gardenlet will be responsible for managing it (according to its `seedSelector` configuration).
+`disable-capacity-reservation` | set `spec.settings.excessCapacity.enabled` in Seed to false (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`invisible` | set `spec.settings.scheduling.visible` in Seed to false  (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`visible` | set `spec.settings.scheduling.visible` in Seed to true  (see [/example/50-seed.yaml](../../example/50-seed.yaml)) **default**
+`disable-dns` | set `spec.settings.shootDNS.enabled` in Seed to false  (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`protected` | only Sshoots in the `garden` namespace can use this seed
+`unprotected` | Shoots from all namespaces can use this seed **default**
+`with-secret-ref` | creates a secret with the kubeconfig of the cluster in the `garden` namespace in the garden cluster and specifies the `.spec.secretRef` in the `Seed` object accordingly.
+`shootDefaults.pods` | default pod network CIDR for shoots created on this seed
+`shootDefaults.services` | default service network CIDR for shoots created on this seed
+`minimumVolumeSize` | set `spec.volume.minimumSize` in Seed (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`blockCIDRs` | set `spec.network.blockCIDRs` seperated by `;` (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`backup.provider` | set `spec.backup.provider` in Seed (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`backup.region` | set `spec.backup.region` in Seed (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`backup.secretRef.name` | set `spec.backup.secretRef.name` in Seed (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`backup.secretRef.namespace` | set `spec.backup.secretRef.namespace` in Seed (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`apiServer.autoscaler.minReplicas` |
```

Comment:
If there is no description,  the `|` could be omitted?

Reply:
I asked rfranzke to add a description for this options. I don`t know a description for that.